metricflow/test/query/test_query_parser.py

```diff
@@ -256,13 +252,12 @@ def test_query_parser_case_insensitivity(bookings_query_parser: MetricFlowQueryP
 
 
 def test_query_parser_invalid_group_by(bookings_query_parser: MetricFlowQueryParser) -> None:  # noqa: D
-    with pytest.raises(UnableToSatisfyQueryError):
+    with pytest.raises(InvalidQueryException):
```

Comment:
We probably need to communicate this out so MFS/SLG exception handling for alerts and stuff accounts for this in the same way they account for UnableToSatisfyQueryError.